tests/test_views/test_flags.py

```diff
@@ -0,0 +1,110 @@
+"""Test flags api endpoints."""
+
+from unittest.mock import Mock, patch
+
+from bson import ObjectId
+from django.test import Client
+
+from forum.models.contents import Contents
+from forum.models.users import Users
+
+
+def test_comment_thread_api(api_client: Client, users_model: Users, content_model: Contents) -> None:
+    """
+    Test the comment thread flag API.
+
+    This test checks that a user can flag a comment thread for abuse and then unflag it.
+    """
+    user_id = users_model._collection.insert_one({"_id": "1"}).inserted_id
```

Comment:
Use insert instead of using  _collection.insert_one.

Reply:
Done